data/transaction.go

```diff
@@ -122,4 +120,12 @@ type PreparedResults struct {
 	ScResults       []*ScResult
 	Receipts        []*Receipt
 	AlteredAccounts map[string]*AlteredAccount
+	DeploysInfo     []*ScDeployInfo
+}
+
+// ScDeployInfo is the TDO that holds information about a smart  contract deploy
```

Comment:
extra space, DTO instead of TDO
...about a smart contract deployment

Reply:
done.